models.js

```diff
@@ -17,10 +17,10 @@ exports.fetchArticles = () => {
 
   return db.query(queryText).then(({ rows }) => {
     rows.map((article) => {
-        article.comment_count = Number(article.comment_count)
-        return article
-    })
-    return rows
+      article.comment_count = Number(article.comment_count);
```

Comment:
You can alter the data type in the sql query rather than using javascript for it. Have a look at casting :)